amf-api-contract/shared/src/main/scala/amf/apicontract/internal/spec/async/parser/bindings/server/IBMMQServerBindingParser.scala

```diff
@@ -0,0 +1,28 @@
+package amf.apicontract.internal.spec.async.parser.bindings.server
+
+import amf.apicontract.client.scala.model.domain.bindings.ibmmq.IBMMQServerBinding
+import amf.apicontract.internal.metamodel.domain.bindings.IBMMQServerBindingModel
+import amf.apicontract.internal.spec.async.parser.bindings.BindingParser
+import amf.apicontract.internal.spec.async.parser.context.AsyncWebApiContext
+import amf.core.internal.parser.YMapOps
+import amf.core.internal.parser.domain.Annotations
+import org.yaml.model.{YMap, YMapEntry}
+
+object IBMMQServerBindingParser extends BindingParser[IBMMQServerBinding] {
+  override def parse(entry: YMapEntry, parent: String)(implicit ctx: AsyncWebApiContext): IBMMQServerBinding = {
+    val binding = IBMMQServerBinding(Annotations(entry))
+    val map     = entry.value.as[YMap]
+
+    map.key("groupId", IBMMQServerBindingModel.GroupId in binding)
```

Comment:
Watch out because most of these facets have default values.

Reply:
done, now we're parsing the value or setting the default one where applicable